LayoutTests/fast/events/touch/ios/content-observation/add-mouseout-event-listener-on-hover.html

```diff
@@ -8,6 +8,7 @@
 jsTestIsAsync = true;
 expandedContainer = false;
 window.internals?.settings.setContentChangeObserverEnabled(true);
+window.internals?.setTopDocumentURLForQuirks("https://store.steampowered.com");
```

Comment:
Should we reflect this change in the file name too? I don't know if that's the norm for quirky tests.

Reply:
Makes sense! I'll add a `-quirk` suffix to the test.